### PR TITLE
🧑🏻‍🌾 `1.98` cherries

### PR DESCRIPTION
**Pull request by Duciwuci**
**How to categorize this PR?**

/kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**NEW**
Added https://github.com/gardener/gardener/pull/10442
* Can be removed with Gardener `v1.100`

**DROPPED**
According to PR https://github.com/stackitcloud/ske-base/pull/1663 we can drop:
* Use SKE etcd-druid: https://github.com/stackitcloud/gardener/commit/2c2c0458d40e615966472d767d706bebb54e5219
* Allow changing backup provider to S3: https://github.com/stackitcloud/gardener/commit/031cabb4a580bcf68a3f3495d7f24a82ad18b24d

Also dropped https://github.com/stackitcloud/gardener/commit/b10499c8c928f7b0f37d729c69b2a493bf35d78a

Dropped https://github.com/stackitcloud/gardener/pull/97/commits/d26e32dd2fd0393315103053e65b953868e1a35f because it is included in `v1.98`

### CODE REVIEW

**Comment by Duciwuci** [comment]
Currently https://github.com/stackitcloud/gardener/pull/104/commits/6c1dc976c245f289ebc6c42af244ba891acfea14 causes failing integration tests. Must be fixed before continuoing.

File: pkg/component/networking/apiserverproxy/apiserver_proxy_test.go [hunk 1]
```diff
@@ -462,6 +462,12 @@ static_resources:
           "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
           stat_prefix: kube_apiserver
           cluster: kube_apiserver
+		  tunneling_config:
```
Comment (Duciwuci) [inline pkg/component/networking/apiserverproxy/apiserver_proxy_test.go:465]: This not to seem correct: https://github.com/stackitcloud/gardener/pull/97/commits/a57ce23b3b16917161593165f55f543fd9beb88f#diff-08f1f63ed2f60c43bf365ed0febad215ee59b84095021725dceb61d95c878469

**Comment by nschad** [comment]
Please also drop commit: b10499c8c928f7b0f37d729c69b2a493bf35d78a

**Comment by Duciwuci** [comment]
@nschad done <3

**Review by nschad — CHANGES_REQUESTED** [review]
@Duciwuci Please **drop** the commit. Do not revert it!. This will PR will rebase merged, that would show up.

**Comment by Duciwuci** [comment]
There is no need to wait

**Pull request merged**